### PR TITLE
Schedule oilvap

### PR DESCRIPTION
**Pull request by joakim-hove**
Downstream: 
https://github.com/OPM/opm-material/pull/437
https://github.com/OPM/opm-simulators/pull/3007

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this opm-material=437 opm-simulators=3007 please

**Comment by joakim-hove** [comment]
jenkins build this opm-material=437 opm-simulators=3007 please

File: src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp [hunk 1]
```diff
@@ -239,9 +239,9 @@ namespace {
         for (const auto& record : handlerContext.keyword) {
             std::fill(max.begin(), max.end(), record.getItem("DRSDT_MAX").getSIDouble(0));
             std::fill(options.begin(), options.end(), record.getItem("Option").get< std::string >(0));
-            OilVaporizationProperties ovp = this->m_oilvaporizationproperties.get(handlerContext.currentStep);
+            OilVaporizationProperties ovp = handlerContext.sched_step.oilvap();
             OilVaporizationProperties::updateDRSDT(ovp, max, options);
-            this->m_oilvaporizationproperties.update( handlerContext.currentStep, ovp );
+            handlerContext.sched_step.oilvap( ovp );
```
Comment (bska) [inline src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp:244]: Is there a reason we're not using `this->snapshots.back()` here?

Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp:244]: Rebase rebase .... 😟

**Comment by joakim-hove** [comment]
jenkins build this opm-material=437 opm-simulators=3007 please

**Comment by joakim-hove** [comment]
jenkins build this opm-material=437 opm-simulators=3007 please

**Pull request merged**